ddocs/medic-db/medic-client/views/reports_by_freetext/map.js

```diff
@@ -15,32 +18,33 @@ function(doc) {
     if (!key || !value) {
       return;
     }
-    key = key.toLowerCase();
-    if (skip.indexOf(key) !== -1 || /_date$/.test(key)) {
-      return;
-    }
+
     if (typeof value === 'string') {
       value = value.toLowerCase();
       value.split(/\s+/).forEach(function(word) {
         emitMaybe(word, reportedDate);
       });
     }
-    if (typeof value === 'number' || typeof value === 'string') {
-      emitMaybe(key + ':' + value, reportedDate);
-    }
   };
 
-  if (doc.type === 'data_record' && doc.form) {
-    Object.keys(doc).forEach(function(key) {
-      emitField(key, doc[key], doc.reported_date);
-    });
+  var include = ['patient_id', 'patient_uuid', 'place_id', 'case_id', 'patient_name'];
+  include.forEach(function(key) {
+    emitField(key, doc[key], doc.reported_date);
+
     if (doc.fields) {
-      Object.keys(doc.fields).forEach(function(key) {
-        emitField(key, doc.fields[key], doc.reported_date);
-      });
-    }
-    if (doc.contact && doc.contact._id) {
-      emitMaybe('contact:' + doc.contact._id.toLowerCase(), doc.reported_date);
+      emitField(key, doc.fields[key], doc.reported_date);
     }
+  });
+
+  if (doc.contact && doc.contact._id) {
+    emitMaybe('contact:' + doc.contact._id.toLowerCase(), doc.reported_date);
```

Comment:
This is an interesting case I had not noticed before....  :thinking: [Looks like](https://github.com/medic/cht-core/commit/b4cd9a541ee076e140700fe9302ae043f220692b) this is to support the ancient ANC workflow Gareth mentioned [here](https://github.com/medic/cht-core/issues/9293#issuecomment-2311658854).  I cannot see any place this is currently used in the code (but after my adventures with the `case_id`, who knows...).  

I guess maybe it is safer to leave it... :shrug: